rules/opentitan_test.bzl

```diff
@@ -222,6 +230,7 @@ def opentitan_functest(
         test_in_rom = False,
         ot_flash_binary = None,
         signed = False,
+        slot = "silicon_creator_a",
```

Comment:
nit: can we make "virtual" (the setting that we will use most until ePIC is ready) the default?

Reply:
sure thing! done.

Reply:
actually (as we discussed offline) need to keep the default slot to the virtual slot, b/c address translation is not enabled in the test ROM yet. once it is added, we can make the default slot the virtual slot. Tracking in #10712